scripts/nuget_publish.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# nuget_publish.sh
+# A local version of the fluentui-apple-publish-nuget.yml Azure Pipeline that fires all the jobs sequentially.
+# Useful for local validation, keep in sync with .ado/fluentui-apple-publish-nuget.yml
+# Note: Execute this script from the root of the repository, such as `./scripts/nuget_publish.sh`
+
+# Keep an exit code so that we can return a non-zero exit code if any build fails
+# while still running all builds each time.
+EXIT_CODE=0
+
+# Path to the xcode build wrapper script
+XCODEBUILD_WRAPPER_LOCATION='scripts/xcodebuild_wrapper.sh'
+
+# Extra arguments for xcode build to mimic nuget publishing environment  
+XCODEBUILD_EXTRA_ARGS='-xcconfig .ado/xcconfig/publish_overrides.xcconfig -derivedDataPath DerivedData'
+
+function handle_exit_code()
```

Comment:
Note: some of this is duplicated with the ci.sh command, but didn't feel worth it to try and deduplicate right now.